readthedocs/projects/tasks.py

```diff
@@ -1008,6 +1022,10 @@ def is_type_sphinx(self):
         """Is documentation type Sphinx."""
         return 'sphinx' in self.config.doctype
 
+    def send_build_status(self, build_pk, state):
+        """Send github build status for pull/merge requests."""
+        send_external_build_status(build_pk, state)
```

Comment:
Do we need this function? Seems we can just call it directly.

Reply:
I was just following Notification pattern to make the code consistent. but I guess we don't need it :)